habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -55,9 +55,13 @@ def __init__(
             discrete_actions = (
                 policy_config.action_distribution_type == "categorical"
             )
+            if not discrete_actions:
+                assert(len(action_space.shape) == 1)
```

Comment:
Should this assertion live on the main repo. Also, I think an error message would be useful after this assert.